app/src/main/java/fr/free/nrw/commons/utils/DialogUtil.java

```diff
@@ -30,9 +30,7 @@ public static void dismissSafely(@Nullable Activity activity, @Nullable DialogFr
             return;
         }
 
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
-            isActivityDestroyed = activity.isDestroyed();
-        }
+        isActivityDestroyed = activity.isDestroyed();
```

Comment:
This variable probably not necessary after refactor

Reply:
@domdomegg This function is not getting used anywhere in the codebase. Is it fine if I remove the whole function altogether? Or should that be done in another PR?